etl/steps/data/garden/un/2023-08-02/comtrade_pandemics.meta.yml

```diff
@@ -0,0 +1,66 @@
+tables:
+  comtrade_pandemics:
+    description: >-
+      Trade value in USD for the imports of different commodities.
+
+      From [UN Stats](https://unstats.un.org/wiki/display/comtrade/Trade+valuation):
+
+
+      Import values are recorded as a CIF-type value. CIF-type values include the transaction value of the goods, the
+      value of services performed to deliver goods to the border of the exporting country, and the value of the services
+      performed to deliver the goods from the border of the exporting country to the border of the importing country.
+
+
+      "Other Asia, not elsewhere specified" refers to territories belonging to Asia, but not specified by country. More details at
+      https://unstats.un.org/wiki/display/comtrade/Taiwan%2C+Province+of+China+Trade+data
+    variables:
+      import_cif_breathing_appliances:
+        title: Imports of breathing appliances and gas masks
+        short_unit: "$"
+        unit: "USD"
+        description: >-
+          Imports of breathing appliances and gas masks; excluding protective masks having neither mechanical parts nor
+          eplaceable filters and excluding apparatus of item no. 9019.20.
+      import_cif_gloves_mittens_mitts_knitted_croacheted:
```

Comment:
I'm curious to know what are you keeping other types of gloves: is it for comparison with the surgical ones?